embeddings/evaluator/sequence_tagging_evaluator.py

```diff
@@ -0,0 +1,13 @@
+from typing import List, Tuple, Dict, Any
+
+from datasets import Metric, load_metric
+
+from embeddings.evaluator.metrics_evaluator import MetricsEvaluator
+
+
+class SequenceTaggingEvaluator(MetricsEvaluator):
+    @property
+    def metrics(self) -> List[Tuple[Metric, Dict[str, Any]]]:
+        return [
+            (load_metric("seqeval"), {}),
```

Comment:
We need to adapt configuration for Evelator as tagset is not correct.

Warnings from `seqeval`
```
seqeval/metrics/sequence_labeling.py:171: UserWarning: impt seems not to be NE tag.
    warnings.warn('{} seems not to be NE tag.'.format(chunk))
```

Reply:
seqeval is compatible with the given scheme of tagging: https://github.com/chakki-works/seqeval